content/2024/06/maxmind-appoints-seasoned-data-science-leader-rupert-young-as-chief-product-officer.md

```diff
@@ -2,12 +2,12 @@
 title: "MaxMind appoints seasoned data science leader Rupert Young as Chief Product Officer"
 date: "2024-06-11"
 images:
-  - /images/2024/06/rupert_blog.png
+  - /images/2024/06/rupert-young-cpo.png
```

Comment:
Now we just need to add this image to the repo
<img width="810" alt="Screenshot 2024-06-11 at 1 18 43 PM" src="https://github.com/maxmind/blog-site/assets/52248962/cba04395-bf81-45d6-a474-9a870ff8b5f1">

Reply:
ah yes i missed the change from png to jpg, my apologies, thank you for being so patient with me, the n00b (LOL is that still a thing...)

Reply:
No worries, I get this stuff is tricky so always happy to help!

Reply:
It looks like `rupert-young-cpo.jpg` still needs to be added to the repo though.